### PR TITLE
V3

### PR DESCRIPTION
**Pull request by scott-wyatt**

### CODE REVIEW

**Comment by scott-wyatt** [comment]
@jaumard I'm not really sure how the original worked... When we pass in the handler as a string, it throws an express error.

**Comment by jaumard** [comment]
maybe trailpack-router has been modified and don't return the same things as v2 :( need to dig more @scott-wyatt

File: test/api/models/Role.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const Model = require('trails/model')
+const Model = require('trails/lib/model')
```
[the remark this answers is not in the document]
Comment (jamrizzi) [reply test/api/models/Role.js]: #65 corrects the bad M

**Comment by jaumard** [comment]
@scott-wyatt all good for me :) take a final look and if it's ok we can merge/deploy this

**Comment by scott-wyatt** [comment]
Yeah, looks like this is still good, we will need to change the appveyor to only run tests on node 8 and above from now on since that is where Trails backwards compatibility is in v3

**Pull request merged**